### ISSUE TITLE
tests: Some errors in standard library doesn't implement xerrors.Wrapper

### ISSUE THREAD

**Issue by Xuanwo** [aos-dev/go-storage#110]
`strconv.NumError` as an exmaple: we need to write following code for unittest:

```go
e := &strconv.NumError{}
ok := errors.As(err, &e)
if ok {
	assert.True(t, errors.Is(e.Err, tt.err))
} else {
	assert.True(t, errors.Is(err, tt.err))
}
```

After go 1.14 released, we can change them into:

```go
assert.True(t, errors.Is(err, tt.err))
```

**Comment by Xuanwo** [comment]
go 1.14 has been released, due to #113 , we will fix this after go 1.15

**Comment by Xuanwo** [comment]
Fixed in #445
